webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java

```diff
@@ -71,7 +71,7 @@
     
     private boolean readonly = false;
     
-    @Column(nullable = true)
+    @Column(nullable = true, length = 128000)
```

Comment:
Marks the column as a "large binary object". As far as I understand, fepending on the DB, such columns are not stored directly in the table, but rather the table contains pointers into a separate LOB storage area. Afaik that is meant to reduce the data that is kept in memory when the table are loaded/scanned.